tool/src/js2/formatter.rs

```diff
@@ -235,9 +235,8 @@ impl<'tcx> JSFormatter<'tcx> {
             hir::PrimitiveType::IntSize(hir::IntSizeType::Usize) => "usize",
             hir::PrimitiveType::Float(hir::FloatType::F32) => "f32",
             hir::PrimitiveType::Float(hir::FloatType::F64) => "f64",
-            hir::PrimitiveType::Int128(_) => {
-                panic!("Javascript backend does not currently support BigInt")
-            }
+            hir::PrimitiveType::Int128(hir::Int128Type::I128) => "i128",
+            hir::PrimitiveType::Int128(hir::Int128Type::U128) => "u128",
```

Comment:
I don't think this is supported, `DiplomatBuf.slice` doesn't currently handle these values, because there's no `Uint128Array`

Reply:
done